app/app/vhosts/new/route.js

```diff
@@ -11,7 +11,7 @@ export default Ember.Route.extend({
     var stack = app.get('stack');
 
     return Ember.RSVP.hash({
-      vhost: this.store.createRecord('vhost', { app }),
+      vhost: this.store.createRecord('vhost', { app, platform: 'alb' }),
```

Comment:
If they all default to `alb` now, we could just set this as the `defaultValue` for the `platform` attr directly on the model.